core/src/main/resources/org/apache/spark/ui/static/historypage-template.html

```diff
@@ -20,17 +20,17 @@
   <thead>
     <tr>
       <th>
-        <span data-toggle="tooltip" data-placement="right" title="ID of the application.">
+        <span data-toggle="tooltip" data-placement="right" title="ID of this application.">
           App ID
         </span>
       </th>
       <th>
-        <span data-toggle="tooltip" data-placement="above" title="Name of the application.">
+        <span data-toggle="tooltip" data-placement="right" title="Name of this application.">
           App Name
         </span>
       </th>
       <th class="attemptIDSpan">
-        <span data-toggle="tooltip" data-placement="above" title="The attempt ID of this application since one application might be launched several times">
+        <span data-toggle="tooltip" data-placement="right" title="The attempt ID of this application since one application might be launched several times">
```

Comment:
![image](https://cloud.githubusercontent.com/assets/26266482/26663268/1b63a3e2-46bc-11e7-829f-b123cdbde41a.png)

The tooltip placement to "above", the prompt position will always be displayed on the APP ID, rather than on the current title.